### PR TITLE
Alsa out

### LINKED ISSUE #31: Audio out driver
As a vocoder dev, I want the audio I generate to be output so that I can hear the fruit of my labour

- Generate audio output

### PR DESCRIPTION
**Pull request by MarkAHarley**
Closes #31

### CODE REVIEW

File: include/alsa_out.hh
[the remark this answers is not in the document]
Comment (MarkAHarley) [reply include/alsa_out.hh:1]: I thought we were doing documentation on seperate issues?

[the remark this answers is not in the document]
Comment (MHCooke) [reply include/alsa_out.hh:1]: Yeah for old code, new code should be documented as it is added

[the remark this answers is not in the document]
Comment (MarkAHarley) [reply include/alsa_out.hh:1]: Which I have done, pls review

**Review by MHCooke — APPROVED** [review]
Looks great! Excited to see it all come together

File: src/alsa_out.cc [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+            }
+
+            // Output audio
+            Audio audio_out = input.queue.front();
+            input.queue.pop();
+            lk.unlock();
+
+            Audio new_audio;
+            for (auto sample : audio_out) {
+                new_audio.push_back(sample);
```
[the remark this answers is not in the document]
Comment (MarkAHarley) [reply src/alsa_out.cc:98]: Need to output the same to each stereo channel.

**Pull request merged**